packages/components/src/SideNavigation/NavigationItem.tsx

```diff
@@ -40,6 +40,8 @@ const ItemContentContainer = styled.div<{ isSelected?: boolean }>`
   flex-flow: row;
   align-items: center;
   padding: 8px 16px 8px 14px;
+  color: ${({ isSelected, theme }) =>
+    isSelected ? theme.colors.copy : theme.colors.grey500};
```

Comment:
Are we still using `colors.copy`?

Reply:
I updated them to be `grey600`, as discussed here with @jpye-finch 
 https://opencrvsworkspace.slack.com/archives/C03MDRPBNJZ/p1664356045550479